config/puma.rb

```diff
@@ -22,3 +22,5 @@
 end
 
 plugin :tmp_restart
+
+set_remote_address(proxy_protocol: :v1) if ENV['PROXY_PROTO_V1']
```

Comment:
Sorry to ask you to change it again, but pretty much all boolean environment variables we use require being explicitly set to `true`. Can you change it to:
```suggestion
set_remote_address(proxy_protocol: :v1) if ENV['PROXY_PROTO_V1'] == 'true'
```